plenum/test/view_change_service/test_view_change_triggered.py

```diff
@@ -80,3 +84,26 @@ def test_view_change_with_next_primary_stopped(looper, txnPoolNodeSet, sdk_pool_
     sdk_ensure_pool_functional(looper, txnPoolNodeSet, sdk_wallet_client, sdk_pool_handle)
     current_view_no = checkViewNoForNodes(txnPoolNodeSet)
     assert current_view_no == old_view_no + 2
+
+
+def test_view_change_with_next_primary_stopped_and_one_node_lost_commit(looper,
+                                                                        txnPoolNodeSet,
+                                                                        sdk_pool_handle,
+                                                                        sdk_wallet_client):
+    old_view_no = checkViewNoForNodes(txnPoolNodeSet)
+    next_primary = get_next_primary_name(txnPoolNodeSet, old_view_no + 1)
+    slow_node = txnPoolNodeSet[-1]
```

Comment:
What if slow_node will be the next primary?
Please, add the check, that slow_node is not the next primary, like:
`delayed_node = [r.node for r in getNonPrimaryReplicas(txnPoolNodeSet) if r.node.name != next_primary][0]`